src/grandorgue/GOOrganController.cpp

```diff
@@ -210,31 +210,28 @@ void GOOrganController::ReadOrganFile(GOConfigReader &cfg) {
   wxString info_filename
     = cfg.ReadFileName(ODFSetting, WX_ORGAN, wxT("InfoFilename"), false);
   wxFileName fn;
+  m_InfoFilename = wxEmptyString;
   if (info_filename.IsEmpty()) {
     /* Resolve organ file path */
     fn = GetODFFilename();
     fn.SetExt(wxT("html"));
     if (fn.FileExists() && !m_FileStore.AreArchivesUsed())
       m_InfoFilename = fn.GetFullPath();
-    else
-      m_InfoFilename = wxEmptyString;
   } else {
-    GOLoaderFilename fname;
-
-    fname.Assign(info_filename);
-    std::unique_ptr<GOOpenedFile> file = fname.Open(m_FileStore);
-    fn = info_filename;
-    if (
-      file->isValid()
-      && (fn.GetExt() == wxT("html") || fn.GetExt() == wxT("htm"))) {
-      if (fn.FileExists() && !m_FileStore.AreArchivesUsed())
+    if (!m_FileStore.AreArchivesUsed()) {
+      GOLoaderFilename::GenerateFullPath(
+        info_filename, wxFileName(GetODFFilename()).GetPath());
+      fn = info_filename;
+      if (
+        fn.FileExists()
+        && (fn.GetExt() == wxT("html") || fn.GetExt() == wxT("htm"))) {
         m_InfoFilename = fn.GetFullPath();
-      else
-        m_InfoFilename = wxEmptyString;
-    } else {
-      m_InfoFilename = wxEmptyString;
-      if (m_config.ODFCheck())
-        wxLogWarning(_("InfoFilename does not point to a html file"));
+      } else {
+        if (m_config.ODFCheck())
+          wxLogWarning(
+            _("InfoFilename %s either does not exist or is not a html file"),
+            fn.GetFullPath());
+      }
     }
   }
```

Comment:
As a result we stop checking the capability of opening the info file. Is this change desirable?

Reply:
See problem description at the top.

Reply:
Yes, I saw.